open-metadata-implementation/access-services/data-platform/data-platform-spring/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/server/spring/DataPlatformOMASResource.java

```diff
@@ -45,11 +46,19 @@ public GUIDResponse createSoftwareServerCapability(@PathVariable("serverName") S
      * @param qualifiedName the qualified name
      * @return the software server capability by qualified name
      */
-    @GetMapping(path = "/software-server-capabilities/{qualifiedName}")
-    public RegistrationRequestBody getSoftwareServerCapabilityByQualifiedName(@PathVariable String serverName,
-                                                                              @PathVariable String userId,
-                                                                              @PathVariable String qualifiedName) {
```

Comment:
I think similar nouns should be used for GET/POST request, meaning registration or softwareServerCapability